rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -124,6 +126,8 @@ def patch_and_enter_before_unlock(
         ksm_rpc_endpoint,
         ethereum_manager_connect_at_start,
         optimism_manager_connect_at_start,
+        polygon_pos_manager_connect_at_start,
```

Comment:
not for here but man we do need to abstract those connection mocks better ... or remove/adjust them